src/renderer/components/node/IteratorNode.tsx

```diff
@@ -163,6 +164,7 @@ const IteratorNode = memo(({ data, selected }: IteratorNodeProps) => {
                     schemaId={schemaId}
                 />
                 <NodeFooter
+                    animated={animated || false}
```

Comment:
This barely works. Using the `||` on nullable booleans is pretty bad practice. Either use `??` or given them a default when destructuring `data` (so `const { ... , animated = false } = data`).

Same for all other occurrences of this pattern.